dir.props

```diff
@@ -71,6 +71,7 @@
   <!-- Enable the analyzers for this repo -->
   <PropertyGroup>
     <EnableDotnetAnalyzers Condition="'$(EnableDotnetAnalyzers)'==''">true</EnableDotnetAnalyzers>
+    <UseWin32Apis>true</UseWin32Apis>
```

Comment:
We were using OSGroup before we should continue to use that instead of OSEnvironment.